frontend/src/pages/LogIn.js

```diff
@@ -17,16 +17,7 @@ function LogInPage() {
         }));
     };
 
-    const handleSubmit = (e) => {
-        e.preventDefault();
-        // future add the code for logic details when ready
-        console.log(formData);
-
-        const DataToSend = {
-            userName: formData.username,
-            password: formData.password,
-        };
-
+    const loginUser = (DataToSend) => {
         fetch('http://localhost:5000/login', {
```

Comment:
All fetch methods need to be made to the new url where the backend is deployed.

So instead of: 

_http://localhost:5000/login_

Do this:

_https://animaldatingapp-backend-nzjce52oiq-ue.a.run.app/login_